earthfile2llb/interpreter.go

```diff
@@ -1587,6 +1589,27 @@ func (i *Interpreter) handleImport(ctx context.Context, cmd spec.Command) error
 	return nil
 }
 
+func (i *Interpreter) handleProject(ctx context.Context, cmd spec.Command) error {
+	projectVal, err := i.expandArgs(ctx, cmd.Args[0], false, false)
+	if err != nil {
+		return i.wrapError(err, cmd.SourceLocation, "failed to expand PROJECT %s", cmd.Args[0])
+	}
+
+	if !strings.Contains(projectVal, "/") {
+		return errors.New("format for PROJECT statement should be: <organization>/<project>")
```

Comment:
We should use `i.errorf` for this. It adds source location metadata to the error.